backend/config/googleAuthConfig.js

```diff
@@ -0,0 +1,40 @@
+const passport = require('passport')
+const User = require('../models/userModel')
+const GoogleOIDCStrategy = require('passport-google-oidc').Strategy;
+
+passport.use(new GoogleOIDCStrategy({
+    clientID: process.env.GOOGLE_CLIENT_ID,
+    clientSecret: process.env.GOOGLE_SECRET,
+    callbackURL: process.env.GOOGLE_REDIRECT,
+},
+async function(accessToken, profile, done){
+    try{
+    //check if user exists in database
+        const user = await User.findOne({email: profile.emails[0].value});
+       
+        if(user){
+            //log the user in
+            done(null,user);
+        }
+        else{
+            const newUser = new User({
+            username: profile.displayName,
+            email: profile.emails[0].value,
+            fName: profile.name.givenName,
+            lName: profile.name.familyName,
+            profilePicture: '',
```

Comment:
should this not be profile.photos[0].value?

Reply:
Yes, you're right! Should I fix this now?

Reply:
Yes. I can approve it and push right away.